
{'date': 'Sat May 26 2018 09:21:44.431 GMt-0300 (-03) -X- SuPyGirls -X-',
'error': '''
 module <string> line 24
  self._esconde
  ^
IndentationError: expected an indented block
'''},
{'date': 'Sat May 26 2018 13:07:41.116 GMt-0300 (-03) -X- SuPyGirls -X-',
'error': '''
 module <string> line 52
  self.t.append('{}µ{}'.format(char,str(window.Date.now())[-5:])
                                                                   ^
SyntaxError: invalid syntax
'''},
{'date': 'Sat May 26 2018 13:08:10.91 GMt-0300 (-03) -X- SuPyGirls -X-',
'error': '''
 module <string> line 52
  self.t.append('{}µ{}'.format(char,str(window.Date().now())[-5:])
                                                                     ^
SyntaxError: invalid syntax
'''},
{'date': 'Sat May 26 2018 13:09:29.605 GMt-0300 (-03) -X- SuPyGirls -X-',
'error': '''
 module <string> line 59
  self.texto = Texto(cena=self.cena, tit="diga,", txt='com q paus a canoa?',
                                                                                                                                                                                  ^
SyntaxError: invalid syntax
'''},
{'date': 'Sat May 26 2018 13:11:12.858 GMt-0300 (-03) -X- SuPyGirls -X-',
'error': '''
 module <string> line 59
  self.texto = Texto(cena=self.cena, tit="diga,", txt='com q paus a canoa?',
                                                                                                                                                                                  ^
SyntaxError: invalid syntax
'''},
{'date': 'Sat May 26 2018 13:15:01.692 GMt-0300 (-03) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 160
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 188
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 360
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 332
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 349
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 70
    Game()
  module <module> line 60
    texto=str("\n".join(self.texto.sai()))
AttributeError: 'Game' object has no attribute 'texto'
'''},
{'date': 'Sat May 26 2018 13:19:18.924 GMt-0300 (-03) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 160
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 188
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 360
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 332
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 349
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 69
    Game()
  module <module> line 64
    self.texto.vai()
  module <module> line 39
    self.cena.elt <= Popup.POP.popup
NameError: name 'Popup' is not defined
'''},
{'date': 'Sat May 26 2018 13:19:39.933 GMt-0300 (-03) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 160
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 188
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 360
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 332
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 349
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 69
    Game()
  module <module> line 64
    self.texto.vai()
  module <module> line 39
    self.cena.elt <= Popup.POP.popup
NameError: name 'Popup' is not defined
'''},
{'date': 'Sat May 26 2018 13:19:40.920 GMt-0300 (-03) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 160
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 188
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 360
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 332
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 349
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 69
    Game()
  module <module> line 64
    self.texto.vai()
  module <module> line 39
    self.cena.elt <= Popup.POP.popup
NameError: name 'Popup' is not defined
'''},
{'date': 'Sat May 26 2018 13:24:02.322 GMt-0300 (-03) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 160
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 188
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 360
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 332
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 349
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 69
    Game()
  module <module> line 64
    self.texto.vai()
  module <module> line 39
    self.cena.elt <= Popup.POP.popup
NameError: name 'Popup' is not defined
'''},
{'date': 'Sat May 26 2018 13:25:21.795 GMt-0300 (-03) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 160
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 188
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 360
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 332
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 349
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 69
    Game()
  module <module> line 64
    self.texto.vai()
  module <module> line 39
    self.cena.elt <= Popup.POP.popup
NameError: name 'Popup' is not defined
'''},
{'date': 'Sat May 26 2018 13:29:23.664 GMt-0300 (-03) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 160
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 188
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 360
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 332
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 349
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 69
    Game()
  module <module> line 64
    self.texto.vai()
  module <module> line 39
    self.cena.elt <= Popup.POP.popup
NameError: name 'Popup' is not defined
'''},